### PR TITLE
Scheduled scrub scenarios

### PR DESCRIPTION
**Pull request by SrinivasaBharath**
# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [x] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [x] Review the automation design
- [x] Implement the test script and perform test runs
- [x] Submit PR for code review and approve
- [x] Update Polarin Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by SrinivasaBharath** [comment]
The test case execution Log files- http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-A2CLOI

File: suites/pacific/rados/tier_3_rados_scrub.yaml [hunk 1]
```diff
@@ -70,3 +70,32 @@ tests:
...
+      config:
+        scenario: "beginTime and endTime gt currentTime"
+
+  - test:
+      name: Begin Time & End time > current
+      polarion-id: CEPH-9368
+      module: scheduled_scrub_scenarios.py
+      desc: Scheduled scrub validation
+      config:
+        scenario: "beginTime and endTime gt currentTime"
```
Comment (pdhiran) [inline suites/pacific/rados/tier_3_rados_scrub.yaml:87]: This and the above scenario ( ` beginTime and endTime gt currentTime `) appear to be same with different polarion ID's. Is that intended?

Comment (SrinivasaBharath) [reply suites/pacific/rados/tier_3_rados_scrub.yaml:87]: Looking into it.

Comment (SrinivasaBharath) [reply suites/pacific/rados/tier_3_rados_scrub.yaml:87]: The scenario which it missed is  B & E < C where 
B= Begin Time
E = End Time
C= Current Time
This scenario I am covering in the test case where B>E and C< E. In this, both B & E are less than C but B >E.
I will modify it according to this.

File: ceph/rados/rados_scrub.py [hunk 2]
```diff
@@ -158,3 +158,21 @@ def add_begin_end_hours(self, begin_hours=0, end_hours=0):
             cluster_end_hour,
             cluster_end_weekday,
         )
+
+    def set_osd_flags(self, flag, value):
+        """
+        Used to set/unset the osd flags
+
+        Args:
+            1.flag: set or unset the flags
```
Comment (pdhiran) [inline ceph/rados/rados_scrub.py:167]: Can we add examples of allowed values?

Comment (SrinivasaBharath) [reply ceph/rados/rados_scrub.py:167]: I will add the example in the next PR which I need to raise after this merge.

**Review by pdhiran — APPROVED** [review]
Looks good

**Review by psathyan — APPROVED** [review]

**Pull request merged**